bound/bound_test.go

```diff
@@ -126,3 +126,50 @@ func TestUnion(t *testing.T) {
 func same(a, b Bound) bool {
 	return floats.Same([]float64{a.Min, a.Max}, []float64{b.Min, b.Max})
 }
+
+var jaccardTests = []struct {
+	bounds []Bound
+	want   float64
+}{
+	{
+		bounds: []Bound{{Min: 0, Max: 5}},
+		want:   1.0,
```

Comment:
Note that the decimal and trailing zero is not needed since `want` is already declared to be a `float64`.